airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1698,7 +1698,9 @@ definitions:
       page_size:
         title: Page Size
         description: The number of records to include in each pages.
-        type: integer
+        anyOf:
+          - type: integer
+          - type: string
         examples:
```

Comment:
can you add an interpolation_context to document that the config is available?

example:
```
        interpolation_context:
          - config
```

Reply:
Updated.